README.md

```diff
@@ -111,11 +111,12 @@ Sweep interval configures how often to scan for expired rate limiters. The defau
 
 ### Caddyfile config
 
-As with all non-standard HTTP handler modules, this directive is not known to the Caddyfile adapter and so it must be ["ordered"](https://caddyserver.com/docs/caddyfile/directives#directive-order) manually using global options unless it only appears within a [`route` block](https://caddyserver.com/docs/caddyfile/directives/route). This ordering usually works well, but you should use discretion:
+By default, the `rate_limit` directive is ordered before `basic_auth` in the Caddyfile. This simplifies configuration and removes the need for manual ordering in most cases. 
+However, if this default order does not suit your needs, you can still change the order using the [`order`](https://caddyserver.com/docs/caddyfile/directives#directive-order) global directive. For example:
```

Comment:
We should restore mention of the `route` directive since it can be useful too.

Reply:
I have restored the mention of `route` and added an example for this as well.